samples/Cryptocash.Api/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Application.Commands.CreateRefBookingToBookingFeesForCommissionCommand.g.cs

```diff
@@ -0,0 +1,59 @@
+﻿
+// Generated
+
+#nullable enable
+
+using MediatR;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Nox.Application;
+using Nox.Application.Commands;
+using Nox.Factories;
+using Nox.Solution;
+using Nox.Types;
+
+using Cryptocash.Infrastructure.Persistence;
+using Cryptocash.Domain;
+using Cryptocash.Application.Dto;
+
+namespace Cryptocash.Application.Commands;
+public record CreateRefBookingToBookingFeesForCommissionCommand(BookingKeyDto EntityKeyDto, CommissionKeyDto RelatedEntityKeyDto) : IRequest <bool>;
+
+public partial class CreateRefBookingToBookingFeesForCommissionCommandHandler: CommandBase<CreateRefBookingToBookingFeesForCommissionCommand, Booking>, 
```

Comment:
small change that you can do in a following PR, create a base class to hold all the methods (the handle in this case), and mark them as virtual so users can override them

You can see the other commands